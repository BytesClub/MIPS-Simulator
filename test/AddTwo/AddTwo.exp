
Sum of two numbers:
35