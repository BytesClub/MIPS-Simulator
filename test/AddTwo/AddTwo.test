
Sum of two numbers:
33