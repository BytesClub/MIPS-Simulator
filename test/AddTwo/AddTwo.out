{"SyntaxTree":[{"index":22,"item":{"action":"Load","type":"Integer","arguments":2,"argument":["$s0","12"],"argumentType":[]}},{"index":23,"item":{"action":"Load","type":"Integer","arguments":2,"argument":["$s1","23"],"argumentType":[]}},{"index":24,"item":{"action":"Add","type":"Register","arguments":3,"argument":["$s3","$s1","$s0"],"argumentType":[]}},{"index":25,"item":{"action":"Load","type":"Integer","arguments":2,"argument":["$v0","4"],"argumentType":[]}},{"index":26,"item":{"action":"Load","type":"Address","arguments":2,"argument":["$a0","out_string"],"argumentType":[]}},{"index":27,"item":{"action":"OS","type":"Interrupt","arguments":0,"argument":[]}},{"index":28,"item":{"action":"Load","type":"Integer","arguments":2,"argument":["$v0","1"],"argumentType":[]}},{"index":29,"item":{"action":"Move","type":"Register","arguments":2,"argument":["$a0","$s3"],"argumentType":[]}},{"index":30,"item":{"action":"OS","type":"Interrupt","arguments":0,"argument":[]}},{"index":31,"item":{"action":"Load","type":"Integer","arguments":2,"argument":["$v0","10"],"argumentType":[]}},{"index":32,"item":{"action":"OS","type":"Interrupt","arguments":0,"argument":[]}}],"SymbolTable":[{"name":"out_string","type":"Data Declaration","value":{"encoding":".asciiz","data":"\nSum of two numbers:\n"}},{"name":"main","type":"Program Instructions","value":3}]}